open_humans/templates/member/activity-management.html

```diff
@@ -171,7 +171,7 @@ <h3>Project has changed permissions!</h3>
     <p><small>
       You can withdraw and deauthorize this project at any time to
       immediately revoke all granted permissions.
-      <a href="{{ disconnect_url }}?next={{ request.path }}">
```

Comment:
This isn't a login/signup process so I don't think the `next` should be messed with?